packages/core/src/helpers/template.helper.ts

```diff
@@ -22,14 +56,58 @@ export const html = litteralTemplate;
  * `;
  * @returns The concatenated CSS string
  */
-export const css = litteralTemplate;
-
-function litteralTemplate(
+export const css = (
   strings: TemplateStringsArray,
   ...args: Array<string | number | boolean>
-): string {
+): string => {
   return strings.reduce(
-    (acc, curr, index) => acc + curr + (args[index] ?? ''),
+    (acc, curr, index) => `${acc}${curr}${args[index] ?? ''}`,
     ''
   );
+};
+
+function createMarker(node: Node | Array<Node>, i: number): HtmlMarkupResult {
+  const key = `marker-${i}`;
+
+  return Array.isArray(node)
+    ? node.reduce(
+        (acc, n, j) => ({
```

Comment:
Could you replace i by "k1" and j by "k2", since their purpose is to create key names?